README.md

```diff
@@ -65,7 +65,7 @@ Pre-release screenshots:
 ```
 git clone --recursive https://github.com/korcankaraokcu/PINCE
 cd PINCE
-sudo sh install_pince.sh
+sh install_pince.sh
```

Comment:
Since we now use a virtual environment for pip and python 3, I don't think we should run the installer as sudo anymore to avoid installing these pip packages as admin.

The script will ask for the password when sudo installing the non-pip packages required by the installer anyway.